docs/api/model.train.rst

```diff
@@ -26,7 +26,7 @@ Word Embeddings
     :nosignatures:
 
     EmbeddingModel
-    SimpleEmbeddingModel
+    CSREmbeddingModel
```

Comment:
May want to change to name like https://pytorch.org/docs/stable/nn.html#torch.nn.EmbeddingBag